src/client/theme-default/styles/vars.css

```diff
@@ -262,20 +262,14 @@
 
 :root {
   --vp-font-family-base:
-    'Inter', ui-sans-serif, system-ui, sans-serif, 'Apple Color Emoji',
+    'Inter', -apple-system, BlinkMacSystemFont, sans-serif, 'Apple Color Emoji',
     'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
   --vp-font-family-mono:
     ui-monospace, 'Menlo', 'Monaco', 'Consolas', 'Liberation Mono',
     'Courier New', monospace;
   font-optical-sizing: auto;
 }
 
-:root:where(:lang(zh)) {
-  --vp-font-family-base:
-    'Punctuation SC', 'Inter', ui-sans-serif, system-ui, sans-serif,
```

Comment:
Why was this removed? I think it's still needed because otherwise the punctuation will use Inter instead of Chinese font. 👀

Reply:
Yes, these - https://github.com/vuejs/vitepress/blob/7658ad35003409e3faa95793f88d3df4147d6dc9/src/client/theme-default/styles/fonts.css#L157-L159